cluster-autoscaler/utils/drain/drain_test.go

```diff
@@ -223,6 +223,41 @@ func TestDrain(t *testing.T) {
 		},
 	}
 
+	zeroGracePeriod := int64(0)
+	unknownStatePodWithDeletionTimestamp := &apiv1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:              "bar",
+			Namespace:         "default",
+			DeletionTimestamp: &metav1.Time{Time: time.Now().Add(-1 * PodLongTerminatingExtraThreshold)},
+			OwnerReferences:   GenerateOwnerReferences(rc.Name, "ReplicationController", "core/v1", ""),
+		},
+		Spec: apiv1.PodSpec{
+			NodeName:                      "node",
+			RestartPolicy:                 apiv1.RestartPolicyOnFailure,
+			TerminationGracePeriodSeconds: &zeroGracePeriod,
+		},
+		Status: apiv1.PodStatus{
+			Phase: apiv1.PodUnknown,
+		},
+	} // TODO test with 40s and default TerminationGracePeriod
```

Comment:
Could you either add that test case or remove the TODO?

Reply:
ok